src/math/ChebyshevBoundedField.cc

```diff
@@ -313,6 +313,30 @@ double ChebyshevBoundedField::evaluate(geom::Point2D const & position) const {
                               p.getY(), _coefficients.getSize<0>());
 }
 
+// The integral of T_n(x) over [-1,1]:
+// https://en.wikipedia.org/wiki/Chebyshev_polynomials#Differentiation_and_integration
+double integrateTn(int n) {
+    if (n % 2 == 1)
+        return 0;
+    else
+        return 2./(1. - double(n*n));
+}
+
+double ChebyshevBoundedField::integrate() const {
+    double result = 0;
+    double determinant = getBBox().getArea() / 4.;
```

Comment:
Coding standards for C++ say we should use `4.0`, not `4.` (I think we're free to do either in Python, formally, but I think it'd be best to use the same formatting there, too).